web/src/features/charts/BreakdownChart.tsx

```diff
@@ -29,6 +31,11 @@ function BreakdownChart({
 
   const { chartData, valueAxisLabel, layerFill, layerKeys } = data;
 
+  // Find highest daily emissions to show correct unit on chart
+  const maxEmissions = max(chartData.map((day) => sum(Object.values(day.layerData))));
```

Comment:
I think this should use absolute values (all scaling should) so we don't risk exports being wrong.

Reply:
Not sure I understand what you mean here.
This is used to make sure we show "0t" (or whatever unit the highest value on the chart is) instead of "0g" at the beginning of the y-axis.

Reply:
Albeit an edge case if say a zone has 3 modes that are each producing 500kg in emissions and then a export/storage of 1t (1000kg) then the unit should probably be 0t and not 0kg.

Might not matter much since we aren't showing negative values (yet) but if nothing has changes the plan is still to show storage charge and exports in the breakdown graph right?

Reply:
Hmmmm, wouldn't it be weird if this graph included exports/storage? At last currently it's showing the "origin of emissions", so not accounting for what's exported. How would you show both in a nice and not confusing way? 🤔 

<img width="448" alt="Screenshot 2023-08-30 at 07 58 47" src="https://github.com/electricitymaps/electricitymaps-contrib/assets/3296643/b7f1a4e7-3845-4eee-8b30-3873be9c77d8">

Anyway, sounds like a future discussion and I suggest we wait with optimizing the units for this until then :)

Reply:
We have discussed it a while ago related to this issue: https://github.com/electricitymaps/electricitymaps-contrib/issues/5259

But perhaps we need to rethink the implementation?

Reply:
Oh sorry, I forgot all about that! Let's keep it as a separate discussion, I'd like to take a step back and try making a few examples (in Figma, Paint, or whatever) to discuss usability if we change and what makes sense for users :)